lesson6/hometask_frameq.rb

```diff
@@ -0,0 +1,87 @@
+require 'YAML'
+
+class Person
+
+  attr_accessor :age, :sex, :fullname, :file, :name
+
+  def self.add(name, &block)
+    self.class.send(:attr_accessor, name)
```

Comment:
Hm. There is one bad thing here. You are setting new accessor for **all instances of Class class**. Try to put `String.one` just after this line. You will get `nil` result, but should get en exception `*** NoMethodError Exception: undefined method 'one' for String:Class`.
You can resolve this by replacing `class` to `singleton_class`. In that case you will add accessor only for metaclass of `Person`, not for all instances of Class class.